src/magick-image.ts

```diff
@@ -552,6 +553,24 @@ export class MagickImage extends NativeInstance implements IMagickImage {
 
     get width(): number { return ImageMagick._api._MagickImage_Width_Get(this._instance); }
 
+    addProfile(name:string, data:Uint8Array) {
```

Comment:
In the rest of the code there is a space after the `:`.

Reply:
Thanks. I will pay attention next time.